processing_r/test/test_translations.py

```diff
@@ -26,12 +26,12 @@ class SafeTranslationsTest(unittest.TestCase):
 
     def setUp(self):
         """Runs before each test."""
-        if 'LANG' in iter(os.environ.keys()):
+        if 'LANG' in iter(os.environ.keys()):  # pylint: disable=consider-iterating-dictionary
```

Comment:
```suggestion
        if os.getenv('LANG'):
            del os.environ['LANG']
```

This should fix the pylint ;-)

My 2 cents, feel free to ignore

or `if 'LANG' in os.environ.keys()` as well.